### PR TITLE
Create a new Reference Architecture tile

### PR DESCRIPTION
**Pull request by haleycode**

### CODE REVIEW

File: content/reference-architecture/cdn-reference-architecture.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+![Figure 1: HTTP Request with no CDN](/reference-architecture/static/cdn-reference-architecture-images/cdn-ref-arch-1.jpg)
+
+## How a CDN tackles web application challenges
+
+A CDN helps address the challenges customers face around latency, performance, availability, redundancy, security, and costs. [A CDN's core goal is to decrease latency and increase performance](https://www.cloudflare.com/learning/cdn/what-is-a-cdn/) for websites and applications by caching content as close as possible to end users or those accessing the content. 
+
+CDNs decrease latency and increase performance by having many data center locations across the globe that cache the content from the origin. The goal is to have content cached as close as possible to users, so content is cached at the edge of the CDN provider's network. 
+
+### The impact this has:
```
Comment (kodster28) [inline content/reference-architecture/cdn-reference-architecture.md]: Headings with colons leading into bulleted lists sorta makes me barf a bit in my mouth...

File: content/reference-architecture/cdn-reference-architecture.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+
+The different cache topologies allow customers to control how Cloudflare interacts with origin servers to help ensure higher cache hit ratios, fewer origin connections, and reduced latency.
+
+| **Smart Tiered Cache Topology (all plans)**                                                                                                                          | **Generic Global Tiered Topology (Enterprise only)**                                                                                                | **Custom Tiered Cache Topology (Enterprise only)**                                                                                                                         |
+|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (kodster28) [inline content/reference-architecture/cdn-reference-architecture.md]: You don't need all these extra lines. Could shorten pretty easily. Let me know if you need help.

File: content/reference-architecture/magic-transit-reference-architecture.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+## 1 Introduction
+
+The purpose of this document is to describe the key architecture, functionalities, and network deployment options of Cloudflare Magic Transit — a BGP-based DDoS protection and traffic acceleration service for Internet-facing network infrastructure. 
+
+## 2 What Is Magic Transit?
+Protecting network infrastructure from DDoS attacks demands a unique combination of strength and speed. Volumetric attacks can easily overwhelm hardware boxes and their bandwidth-constrained Internet links. And most cloud-based solutions redirect traffic to centralized scrubbing centers, which impacts network performance significantly. 
+
+Cloudflare Magic Transit provides DDoS protection and traffic acceleration for on-premise, cloud, and hybrid networks. With data centers spanning 250 cities and with over 100 Tbps in mitigation capacity, Magic Transit can detect and mitigate attacks close to their source of origin in under 3 seconds globally on average — all while routing traffic faster than the public Internet.
+
+![Figure 1: Magic transit overview](/reference-architecture/static/magic-transit-ref-arch-diagrams/magic-transit-ref-arch-1.png)
```
Comment (kodster28) [inline content/reference-architecture/magic-transit-reference-architecture.md:16]: These should always be absolute links, btw, which is what I updated them to.

File: data/reference-architecture.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+name: Reference Architecture
+logo: <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path d="M4 14h22v-4H4v4Zm0 8h22v-4H4v4Zm36 16H26v-4h14v4Zm-14-8h14v-4H26v4Zm34 24H26v-4h34v4Zm-34-8h34v-4H26v4Z"/></svg>
```
Comment (kodster28) [inline data/reference-architecture.yml]: Need to update this to a new SVG (that isn't Zaraz :)  ).

**Review by kodster28 — CHANGES_REQUESTED** [review]

File: content/reference-architecture/magic-transit-reference-architecture.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+
+Of all approaches and deployment options, the always-on method provides the most comprehensive protection. 
+
+However, depending on the provider, diverting all traffic through the DDoS mitigation provider’s cloud might add latency that is suboptimal for business-critical applications. Cloudflare is architected so that customers do not incur a latency penalty as a result of attacks — even for always-on deployments. Analyzing traffic at the edge is the only way to mitigate at scale without impacting performance. 
+
+This is because ingesting traffic via Anycast ensures that traffic travels only to the nearest Cloudflare data center for inspection. With data centers in over 250 cities across more than 100 countries, it is likely to be a short distance. This eliminates the trombone effect. 
+
+In many cases, [traffic is faster when routed over Cloudflare](https://www.cloudflare.com/static/360e550c8890054d5e5835efb9fb8dd1/Magic_Transit_protects_networks_while_also_improving_performance__1_.pdf) than over the public Internet. We believe customers should not have to sacrifice performance to achieve comprehensive security.
```
Comment (kodster28) [inline content/reference-architecture/magic-transit-reference-architecture.md:162]: I'd really prefer not to link to a static asset on the Marketing site and instead just link to the associated marketing page.

This hash will change if the file is updated and -- in all likelihood -- no one will update this link.

Comment (haleycode) [reply content/reference-architecture/magic-transit-reference-architecture.md:162]: Totally hear you, but I'll need to check in with the author before making the change.

File: content/reference-architecture/magic-transit-reference-architecture.md (hunk 3 above)
Comment (haleycode) [reply content/reference-architecture/magic-transit-reference-architecture.md:16]: THANK YOU!

File: data/reference-architecture.yml (hunk 4 above)
Comment (haleycode) [reply data/reference-architecture.yml]: Done!

**Review by kodster28 — APPROVED** [review]

**Pull request merged**